motr/io_req.c

```diff
@@ -1504,8 +1504,9 @@ static int device_check(struct m0_op_io *ioo)
 		node_id = node_obj->pn_id.f_key;
 
 		ti->ti_state = state;
-		if (ti->ti_rc == -ECANCELED) {
-			/* Ignore service failures in a failed node */
+		if (ti->ti_rc == -ECANCELED ||
```

Comment:
Can you also include ENOTCONN here (i have seen that -107 errors are received here, when remote service is stopped) ?

Reply:
@andriytk : ^

Reply:
Yes, will do.